src/frontend/src/app/update-modal/update-modal.component.ts

```diff
@@ -22,23 +22,22 @@ export class UpdateModalComponent implements OnInit, OnChanges {
   @Output() notify: EventEmitter<any> = new EventEmitter<any>();
 
   form: FormGroup;
-  isValidRepeatRange: boolean = false;
-  isValidTimeRange: boolean = false;
   public selectedTypeValue: string;
   constructor(private schedulesService: SchedulesService, public fb: FormBuilder, private alertService: AlertService) { }
 
   ngOnInit() { }
 
   ngOnChanges(changes: SimpleChanges) {
+    let regExp = '^\\d{2}:\\d{2}:\\d{2}'  // Regex to varify time format 00:00:00
     this.form = this.fb.group({
       name: ['', [<any>Validators.required]],
       repeatDay: ['', [<any>Validators.required]],
-      repeat: ['', [<any>Validators.required]],
+      repeat: ['', [<any>Validators.required, Validators.pattern(regExp)]],
       exclusive: [Validators.required],
       process_name: [Validators.required],
       type: [Validators.required],
       day: [Validators.required],
-      time: ['', Validators.required],
+      time: ['', [Validators.required, Validators.pattern(regExp)]],
```

Comment:
using same regexPattern?

Reply:
yes, this regexPattern is only to verify valid entry in time field and in both case it is `00:00:00`